src/tactic/explode_widget.lean

```diff
@@ -14,6 +14,11 @@ occurring in the exploded proofs.
 -/
 open widget tactic tactic.explode
 
+meta def to_string_coe_not {α β} [has_to_string β] : has_coe β (html α) :=
+⟨html.of_string ∘ to_string⟩
+
+local attribute [instance] to_string_coe_not
```

Comment:
Without this, there are several failures below. It seems that https://github.com/leanprover-community/lean/pull/544 is not working properly here.

Reply:
I added a more useful instance here.  Maybe this one should be moved to core though eventually.

Reply:
I was going to suggest adding a TODO comment, but on second thought, this entire file already has "move to core eventually" status.